orchagent/portsorch.cpp

```diff
@@ -2000,18 +2000,44 @@ bool PortsOrch::initializePort(Port &p)
     }
 #endif
 
-    /* Set default port admin status to DOWN */
-    /* FIXME: Do we need this? The default port admin status is false */
-    setPortAdminStatus(p.m_port_id, false);
+    /* Check warm start states */
+    vector<FieldValueTuple> tuples;
+    bool exist = m_portTable->get(p.m_alias, tuples);
+    string adminStatus, operStatus;
+    if (exist)
+    {
+        for (auto i : tuples)
+        {
+            if (fvField(i) == "admin_status")
+            {
+                adminStatus = fvValue(i);
+            }
+            else if (fvField(i) == "oper_status")
+            {
+                operStatus = fvValue(i);
+            }
+        }
+    }
+    SWSS_LOG_INFO("Warm admin_status of %s is %s", p.m_alias.c_str(), adminStatus.c_str());
+    SWSS_LOG_INFO("Warm oper_status of %s is %s", p.m_alias.c_str(), operStatus.c_str());
```

Comment:
what is warm admin_status, can you explain? why not print admin and oper status in the same line?